tests/testthat/testdata/test_configs/config_valid_set_filters_no_data.yaml

```diff
@@ -0,0 +1,89 @@
+schema_version: https://raw.githubusercontent.com/hubverse-org/hubPredEvalsData/main/inst/schema/v1.0.0/config_schema.json
+targets:
+- target_id: wk inc flu hosp
+  metrics:
+  - se_point
+  - ae_point
+  - wis
+  - ae_median
+  - interval_coverage_50
+  - interval_coverage_95
+  disaggregate_by:
+  - location
+  - reference_date
+  - horizon
+  - target_end_date
+eval_sets:
+- eval_set_name: Full season
+  round_filters:
+    min: '2022-10-22'
+- eval_set_name: Last 5 weeks
+  round_filters:
+    min: '2022-10-22'
+    n_last: 5
+- eval_set_name: Valid locations with no data in test hub
+  round_filters:
+    min: '2022-10-22'
+  task_filters:
+    location: ["06", "25"]
+    horizon: [1, 2, 3]
```

Comment:
our test hub includes locations `"06"` and `"25"` in its config file, but doesn't include model outputs for those locations.